esi_leap/tests/api/controllers/test_types.py

```diff
@@ -12,15 +12,35 @@
 
 from esi_leap.api.controllers import types as types
 import unittest as unittest
+from wsme import types as wtypes
+
+
+class Tester(object):
+
+    def __init__(self, name, uuid):
+        self.name = name
+        self.uuid = uuid
 
 
 class TestCollection(unittest.TestCase):
 
     def setUp(self):
         self.test_collection = types.Collection()
         self.test_collection._type = 'stuff'
-        self.test_collection.stuff = [1, 2, 3]
+        self.test_collection.stuff = []
+        self.test_collection.stuff.append(Tester('1', 'aaa'))
+        self.test_collection.stuff.append(Tester('2', 'aaaa'))
+        self.test_collection.stuff.append(Tester('3', 'aaaaa'))
```

Comment:
Using variations of 'aaaa' isn't all that readable. I'd use different characters - aaaaa, bbbbb, ccccc, etc

Reply:
i switched a, b, and c, each of length 5.